gen_mocks.go

```diff
@@ -114,6 +114,35 @@ func (v visitFn) Visit(node ast.Node) ast.Visitor {
 	}
 }
 
+func genFuncDecl(mockTypeName string, methName string, method *ast.FuncType) *ast.FuncDecl {
+	f := &ast.FuncDecl{
+		Recv: &ast.FieldList{List: []*ast.Field{
+			{
+				Names: []*ast.Ident{ast.NewIdent("s")},
+				Type:  &ast.StarExpr{X: ast.NewIdent(mockTypeName)},
+			},
+		}},
+		Name: ast.NewIdent(methName),
+		Type: method,
+		Body: &ast.BlockStmt{},
+	}
+	if method.Results != nil {
```

Comment:
Thanks for submitting this change. If an interface has a method without a return value, the method's mock implementation should still call the stub function in the mock struct (whereas this change makes the mock method a no-op, even when a stub function is defined). Can you update this to generate a mock method body that does that when there is no return value?